src/ucp/api/ucp_def.h

```diff
@@ -744,6 +744,15 @@ typedef struct ucp_ep_params {
      */
     ucp_conn_request_h      conn_request;
 
+    /**
+     * Tracing and analysis tools can identify the endpoint using this name.
+     * To retrieve the endpoint's name, use @ref ucp_ep_query, as the name you
+     * supply may be changed by UCX under some circumstances, e.g. a name
```

Comment:
`as the name that is supplied to @ref ucp_ep_create`

Reply:
I would prefer to keep the same wording for similar items. This description is based on context and worker descriptions (https://github.com/openucx/ucx/pull/6557#discussion_r605124806)